### PR TITLE
Tune for MPI

### PR DESCRIPTION
**Pull request by FG-TUM**
# Description

Tunes the simulation configuration for the new MPI strategy and adds some MPI related features that were not (yet) implemented.

- [x] AltitudeBasedDecomposition Logger
- [x] CSF dependent on local domain size
- [x] Check whether AutoPas configuration is still optimal
  - [x] Changed default to new optimum

## How Has This Been Tested?

- [x] By dilligent Pablo

### CODE REVIEW

File: src/ladds/Simulation.cpp [hunk 1]
```diff
@@ -78,8 +78,9 @@ std::unique_ptr<AutoPas_t> Simulation::initAutoPas(ConfigReader &config, DomainD
   autopas->setVerletRebuildFrequency(verletRebuildFrequency);
   // Scale Cell size so that we get the desired number of cells
   // -2 because internally there will be two halo cells added on top of maxAltitude
-  // FIXME: adapt calculation of CSF to MPI
-  autopas->setCellSizeFactor((maxAltitude * 2.) / ((cutoff + verletSkin) * (desiredCellsPerDimension - 2)));
+  // Assumption: the domain is a cube
+  const auto localDomainLength = domainDecomp.getLocalBoxMax()[0] - domainDecomp.getLocalBoxMin()[0];
+  autopas->setCellSizeFactor(localDomainLength / ((cutoff + verletSkin) * (desiredCellsPerDimension - 2)));
```
Comment (gomezzz) [inline src/ladds/Simulation.cpp:83]: Not sure how autopas reacts to desiredCellsPerDimension = 2? but this term could become infinite if the user tries

File: src/ladds/Simulation.cpp [hunk 2]
```diff
@@ -78,8 +78,9 @@ std::unique_ptr<AutoPas_t> Simulation::initAutoPas(ConfigReader &config, DomainD
   autopas->setVerletRebuildFrequency(verletRebuildFrequency);
   // Scale Cell size so that we get the desired number of cells
   // -2 because internally there will be two halo cells added on top of maxAltitude
-  // FIXME: adapt calculation of CSF to MPI
-  autopas->setCellSizeFactor((maxAltitude * 2.) / ((cutoff + verletSkin) * (desiredCellsPerDimension - 2)));
+  // Assumption: the domain is a cube
+  const auto localDomainLength = domainDecomp.getLocalBoxMax()[0] - domainDecomp.getLocalBoxMin()[0];
```
Comment (gomezzz) [inline src/ladds/Simulation.cpp:82]: Do we allow non-cubical domains anywhere? Then we should check 🤔

**Review by gomezzz — APPROVED** [review]
minor questions. tried to build and run , work both 🙃

File: src/ladds/Simulation.cpp (hunk 2 above)
Comment (FG-TUM) [reply src/ladds/Simulation.cpp:82]: We don't specifically check / enforce it but our input file gives no option to do something non-cubic.

File: src/ladds/Simulation.cpp (hunk 1 above)
Comment (FG-TUM) [reply src/ladds/Simulation.cpp:86]: fixed in 77e0c65

**Pull request merged**